wce-ote/wce_tei.js

```diff
@@ -4476,4 +4476,8 @@ var removeSpaceAfterLb=function ($node){
 	};
 
 
-module.exports = addArrows;
```

Comment:
I am wondering if we can just:
```
// for unit tests using node`
if (module) {
   module.exports = addArrows;
}
```
What do you think?

Reply:
I tried that first, it didn't work. I still got errors and it refused to run the ote. I also tried testing for module in different ways and they all failed. The try catch is the only thing I could make work.

Reply:
I was wrong, it does load but it does give console errors. I know lots of packages don't care about those. If you think the error is okay then you can change it.